src/app.js

```
@@ -1 +1,72 @@
 // write code here
+const fs = require('fs');
+
+const path = require('path');
+
+function moveFile(sourcePath, destinationPath) {
+  try {
+    if (!fs.existsSync(sourcePath)) {
+      console.error(`Error: Source file: ${sourcePath} does not exist.`);
+
+      return;
+    }
+
+    if (!fs.existsSync(destinationPath)) {
+      let destinationFolder = '';
+
+      if (destinationPath.lastIndexOf('/') !== -1) {
+        destinationFolder = destinationPath.substr(
+          0,
+          destinationPath.lastIndexOf('/'),
+        );
+      } else if (destinationPath.lastIndexOf('\\') !== -1) {
+        destinationFolder = destinationPath.substr(
+          0,
+          destinationPath.lastIndexOf('\\'),
+        );
+      }
+
+      if (destinationFolder !== '') {
+        if (!fs.existsSync(destinationFolder)) {
+          console.error(
+            `Error: destination path: ${destinationPath} does not exist and destination folder: ${destinationFolder} does not exist.`,
+          );
+
+          return;
+        }
+      }
```

The task description states: 'In case destination directory does not exist an error must be thrown.' Currently, the code checks if the destination directory exists, and if it doesn't, it prints an error message to the console and returns. Instead of printing to the console, you should throw an error to satisfy the task requirement.